src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/jpa/repository/EmailDomainRepository.kt

```diff
@@ -1,9 +1,13 @@
 package uk.gov.justice.digital.hmpps.externalusersapi.jpa.repository
 
+import org.springframework.data.jpa.repository.Query
 import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.externalusersapi.model.EmailDomain
 import java.util.UUID
 
 interface EmailDomainRepository : CrudRepository<EmailDomain, UUID> {
   fun findByName(name: String): EmailDomain?
+
+  @Query(value = "SELECT * FROM email_domain WHERE :domainName like name ", nativeQuery = true)
+  fun findByNameLike(domainName: String): EmailDomain?
 }
```

Comment:
I did this as a native query as we are trying to search on a field that contains a "%" and the jpa call kept escaping the column value. 
Note.... the like check is the opposite way round to what you would normally expect as the stored value holds a "%"
e.g. select * from email_domain where 'fred.smith.co.uk' like '%smith.co.uk'